semantics_analysis/relation_extractor/ontology_utils.py

```diff
@@ -704,9 +704,9 @@
                 'description': 'модель не создана в указанной организации',
                 'examples': [
                     {
-                        'text': 'Модель обучали на наборе данных из более чем 7,5 млрд предложений как из базы Facebook, так и из других источников.',
-                        'Model': 'Модель',
-                        'Organization': 'Facebook'
+                        'text': 'Используя новый алгоритм упаковки, в Graphcore ускорили обработку естественного языка более чем в 2 раза при обучении BERT-Large.',
+                        'Model': 'BERT-Large',
+                        'Organization': 'Graphcore'

```

Comment:
This was one of my questions for tomorrow, but still. Can be model trained by someone else except the author? If no, then the example is incorrect indeed.

Reply:
> Can be model trained by someone else except the author?

Yes. E.g. base model can be pretrained in organization A, but then the model can be fine tuned for some specific task in organization B.

But in this case, as I see this, the pretraining is meant, so this can be considered as model creation.